src/io/flutter/view/FlutterView.java

```diff
@@ -237,6 +238,16 @@ protected void perform(AnActionEvent event) {
   }
 }
 
+class ToggleInspectModeAction extends AbstractToggleableAction {
+  ToggleInspectModeAction(@NotNull FlutterView view) {
+    super(view, "Toggle Inspect Mode", "Toggle Inspect Mode", AllIcons.Actions.Menu_find);
+  }
+
+  protected void perform(AnActionEvent event) {
+    view.getFlutterApp().callBooleanExtension("ext.flutter.widgetInspector", isSelected(event));
```

Comment:
Boolean extensions return a bool, and can take an optional bool arg (for example, turning on or off repaint rainbow, or querying whether repaint rainbow is on). I'm wondering if the `ext.flutter.widgetInspector` service protocol extension is a boolean extension, or just a method call to switch into inspector mode.